### ISSUE TITLE
wordpress import error: wrong number of arguments

### ISSUE THREAD

**Issue by khusmann** [jekyll/jekyll-import#387]
Hi! Trying to import wordpress xml with: 

ruby -r rubygems -e 'require "jekyll-import";
    JekyllImport::Importers::WordpressDotCom.run({
      "source" => "wordpress.xml",
      "no_fetch_images" => false,
      "assets_folder" => "assets"
    })'

get error: 
Error: wrong number of arguments (given 2, expected 1) 
/usr/lib/ruby/2.5.0/uri/common.rb:236:in `parse'
/home/<user>/.gems/gems/jekyll-import-0.17.0/lib/jekyll-import/importers/wordpressdotcom.rb:41:in `block in download_images'

when I remove the "allow_redirections" argument from line 41 in wordpressdotcom.rb, everything appears to work.

**Comment by ashmaroli** [comment]
@khusmann To use the patch right away, simply point your `Gemfile` to this repo's `master` branch:
```ruby
# Gemfile

gem "jekyll-import", :git => "https://github.com/jekyll/jekyll-import.git", :branch => "master"
```
and then run using `bundle exec`:
```
bundle exec jekyll import ...
```

**Comment by DirtyF** [comment]
@khusmann the patch has been released in v17.0.1.

**Comment by khusmann** [comment]
Thanks everyone! :)
